rednotebook/util/markup.py

```diff
@@ -265,6 +269,17 @@ def _get_config(target, options):
         config['preproc'].append([r'\\\(\s*(.+?)\s*\\\)', r"BEGINMATH''\1''ENDMATH"])
         config['postproc'].append([r'BEGINMATH(.+)ENDMATH', r'$\1$'])
 
+        # Fix utf8 quotations - „, “ and ” cause problems compiling the latex document
+        config['postproc'].append([u'„', '"'])
+        config['postproc'].append([u'”', '"'])
+        config['postproc'].append([u'“', '"'])
+
+        # enabling index
+        config['style'].append('makeidx')
+        config['postproc'].append([r'BEGININDEX(.+?)ENDINDEX', r'\\index{\1}'])
+        config['postproc'].append(['begin{document}', 'makeindex\n\\\\begin{document}'])
+        config['postproc'].append(['end{document}', 'printindex\n\n\\\\end{document}'])
```

Comment:
Can you please add documentation on how to produce a PDF with an index to the "Convert Latex output to PDF" section in `rednotebook/info.py`?